makerpm.pl

```diff
@@ -210,19 +204,26 @@
 noperlpath=1
 nouninstall=1
 nostart=1
+nostop=1
 if [ "\$tempdir" = "" ]; then
 	tempdir=/tmp/.webmin
 fi
-export config_dir var_dir perl autoos port login crypt host ssl nochown autothird noperlpath nouninstall nostart allow atboot makeboot
+export config_dir var_dir perl autoos port login crypt host ssl nochown autothird noperlpath nouninstall nostart nostop allow atboot makeboot
 ./setup.sh >\$tempdir/webmin-setup.out 2>&1
 chmod 600 \$tempdir/webmin-setup.out
 rm -f /var/lock/subsys/webmin
 cd /usr/libexec/webmin
-if [ "\$inetd" != "1" -a "\$startafter" = "1" ]; then
-	/etc/webmin/stop >/dev/null 2>&1 </dev/null
-	/etc/webmin/start >/dev/null 2>&1 </dev/null
-	if [ "\$?" != "0" ]; then
-		echo "warning: Webmin server cannot be restarted. It is advised to restart it manually by\nrunning \\"/etc/webmin/restart-by-force-kill\\" when upgrade process is finished"
+if [ "\$inetd" != "1" ]; then
+	if [ "\$1" == 1 ]; then
+		/etc/webmin/start >/dev/null 2>&1 </dev/null
+		if [ "\$?" != "0" ]; then
+			echo "error: Webmin server cannot be started. It is advised to start it manually\n       by running \\"webmin force-restart\\" command"
```

Comment:
Didn't we agree elsewhere not to recommend using the "webmin" command to control the process? Can this be replace by /etc/webmin/force-restart instead?